test/Chainweb/Test/Pact/PactInProcApi.hs

```diff
@@ -171,9 +173,9 @@ checkBlockTransactions filePrefix bTrans = do
 
     return $ testGroup "BlockTransactions" $ ttTrans : [ttTransHash]
 
-getBlockHeaders :: Int -> [BlockHeader]
-getBlockHeaders n = do
-    let gbh0 = genesis
+getBlockHeaders :: ChainId -> Int -> [BlockHeader]
+getBlockHeaders cid n = do
```

Comment:
I know this isn't your code but can you clean this up? The `do` is driving me crazy.

Reply:
:-) will do